implementations/jsonschema/version.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+
+# Extract the version of the package (boon) from Cargo.toml
```

Comment:
```suggestion
# Extract the version of the package (jsonschema) from uv.lock
```

Reply:
I wonder if `pyproject.toml` would not work better? lock files are generally speaking a mess! haha

Reply:
@tony-go Sure, that's fine if we want to specify the version in the toml file. Right now I haven't. Either way works for me.